docs/references/javascript/organization/invitations.mdx

```diff
@@ -1,15 +1,15 @@
 ---
-title: Organization Invitations
-description: These are all methods on the Organization class that allow you to manage the invitations to an organization.
+title: Organization invitation methods
+description: Learn about the methods on the Organization object that allow you to manage the invitations to an organization.
```

Comment:
```suggestion
description: Learn how to manage the invitations to an organization.
```
I know I haven't brought this up on other PRs, but what do you think of shortening these descriptions? Especially since we're adding "methods" to the title I think the shorter version is clear enough

Reply:
I think "Learn how to manage the invitations to an organization" sounds like it's going to be a guide, and the first description makes it pretty clear that its a reference document that is going to include information about methods.